api/app/controllers/webhook.controller.js

```diff
@@ -0,0 +1,48 @@
+const db = require("../sequelize/models");
+const User = db.user;
+const Review = db.review;
+const Transcript = db.transcript;
+const Op = db.Sequelize.Op;
+
+
+exports.create = async (req, res) => {
+    // const action = req.headers["x-github-event"];
+    const pull_request = req.body;
+     const action = pull_request.action;
+
+
+    // Check if it's a pull request event
+    if (action === 'opened' || action === 'closed' || action === 'synchronize') {
```

Comment:
Is there any other way to tell whether it is a PR event? Then we handle the action accordingly.
Is synchronize a "merge" event?

Reply:
The entire payload is a PR event, in the PR event is where we can check for each action and values